interface/forms/group_attendance/table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS `form_therapy_groups_attendance` (
+  `id` bigint(20) NOT NULL auto_increment,
+  `date` date NOT NULL ,
+  `group_id` int(11),
+  `user` varchar(255) default NULL,
+  `groupname` varchar(255) default NULL,
+  `authorized` tinyint NOT NULL ,
+  `encounter_id` int(11),
+  `activity` tinyint(4) default NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+# enable form to therapy group and not for patient
+UPDATE `registry` SET `category` = 'Clinical' ,`patient_encounter`= 0 , `therapy_group_encounter`= 1 WHERE `directory` = 'group_attendance';
```

Comment:
This is an attendance form and should therefore be accessible only when a group is chosen and not for a regular patient. This form relates to a group's meeting, not to a specific patient.